tests/update_testapps_dependencies.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func main() {
+	var files []string
+
+	root := "./apps"
```

Comment:
I think path format will not work on windows.

Reply:
Moved to .sh file